main/src/main/res/values/strings.xml

```diff
@@ -1980,6 +1980,7 @@
     <string name="onetime_mapthemefixslow_message">Working with non-zipped map themes might decrease map viewer opening performance. You can zip or sync your themes to get faster performance and reduce the delay on starting the map.</string>
     <string name="map_theme_select">Select map theme</string>
     <string name="map_theme_options">Theme Options</string>
+    <string name="map_manage_offline_data">Manage offline data</string>
```

Comment:
This string is already available as `menu_manage_offline_data`, which can be reused as it's for the identical action